src/node.rs

```diff
@@ -40,45 +55,53 @@ impl Default for Sprawl {
 }
 
 impl Sprawl {
+    /// Creates a new [`Sprawl`]
+    ///
+    /// The default capacity of a [`Sprawl`] is 16 nodes.
```

Comment:
Definitely a good idea to include the default capacity here, but we'll have to remember to update this whenever we change it